### PR TITLE
[DOCS] Refresh machine learning rule docs

### PR DESCRIPTION
**Pull request by lcawl**
This PR updates https://www.elastic.co/guide/en/machine-learning/master/ml-configuring-alerts.html

In particular, it:
- Fixes references to "Stack Management > Alerts and Actions", which should now be "Stack Management > Rules"
- Clarifies that only one type of machine learning rule can be created in the ML app
- Removes a screenshot which IMO is unnecessary and therefore doesn't need to be automated
- Refreshes the remaining automated screenshots
- Removes details that are covered in more detail in the Kibana alerting docs, adding links instead.

### CODE REVIEW

**Review by szabosteve — APPROVED** [review]
Thanks for tightening this up! LGTM.

**Comment by szabosteve** [comment]
I'll remove the two screenshots that are removed here from the screenshot automation script in another PR.

**Comment by lcawl** [comment]
> I'll remove the two screenshots that are removed here from the screenshot automation script in another PR.

I re-added the one you'd automated (as well as refreshing the rest of the automated screenshots). Please take another look and see if you like where I've moved it.

File: docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc [hunk 1]
```diff
@@ -30,38 +30,38 @@ ideal for this purpose.
...
+
+In the *{ml-app}* app, you can create only {anomaly-detect} alert rules; create
+them from the {anomaly-job} wizard after you start the job or from the
+{anomaly-job} list.
 
 [[creating-anomaly-alert-rules]]
 === {anomaly-detect-cap} alert
 
-Select the job that the rule applies to.
+When you create an {anomaly-detect-cap} alert rule, you must select the job that
```
Comment (lcawl) [inline docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc:56]:
```suggestion
When you create an {anomaly-detect} alert rule, you must select the job that
```

**Review by darnautov — APPROVED** [review]
LGTM

**Review by szabosteve — APPROVED** [review]
LGTM!

**Pull request merged**